include/libopencm3/stm32/f0/rcc.h

```diff
@@ -374,6 +386,36 @@ Control</b>
 /* API definitions                                                           */
 /*****************************************************************************/
 
+#define RCC_HSI_FREQUENCY			8000000
+#define RCC_HSI48_FREQUENCY			48000000
+
+/*
+ * @define RCC_PLLCFGR_BITS
+ *
+ * This macro packs away bits that are used when configuring the PLL. The
+ * "gotcha" is that PREDIV[3:0] are in CFGR2 rather than CFGR so we pack
+ * those in the bottom 4 bits, knowing we will strip them off and use them
+ * in CFGR2.
+ */
+#define RCC_PLLCFGR_BITS(mul, src, prediv) (\
+		((mul & RCC_CFGR_PLLMUL_MASK) << RCC_CFGR_PLLMUL_SHIFT) |\
+		((src & RCC_CFGR_PLLSRC_MASK) << RCC_CFGR_PLLSRC_SHIFT) |\
+		((src & RCC_CFGR2_PREDIV_MASK) << RCC_CFGR2_PREDIV_SHIFT))
```

Comment:
They are used in the `rcc_pll_clock_setup` in `lib/stm32/f0/rcc.c` because the PLL has two configuration registers in that family. But the API stays the same `rcc_pll_clock_setup(configuration_bits, base_frequency);`